src/monattack.cpp

```diff
@@ -4512,7 +4512,7 @@ bool mattack::grenadier(monster *const z)
     if (z->attitude_to( *target ) == Creature::A_FRIENDLY) {
         return false;
     }
-    int ret = grenade_helper(z, target, 30, 60, grenades);
+    int ret = grenade_helper(z, target, 30, DISTANCE_FROM_CENTER, grenades);
```

Comment:
I suspect this is a range, not a position, it doesn't necessarily need updating.

Reply:
Upon further examination, this was a moves cost.  Returned back to '60'.